### ISSUE TITLE
White Label - Document word]: Luminosity contrast ratio is 4.1:1 of time text, which is less than 4.5:1

### ISSUE THREAD

**Issue by Kusum199** [microsoft/BotFramework-WebChat#5695]
### Is it an issue related to Adaptive Cards?

No.

### What is the PWD impact?

Users with low vision or color-vision deficiencies may have difficulty reading the “Time text” text due to insufficient color contrast, making the form harder to understand, complete, or navigate.

### What browsers and screen readers do this issue affect?

Windows: Edge with Windows Narrator

### Are there any code-based customization done to Web Chat?

No, I am using Web Chat without any customizations except "styleOptions".

### What version of Web Chat are you using?

Latest production

### Which area does this issue affect?

Others or unrelated

### How to reproduce the issue?

1. Open URL: https://compulim.github.io/webchat-loader/ and sign in using valid credentials.
2. Navigate to web chat page and select 4.18.1-main.20240927.ea7a875 (30/9/2024) version and then select mockBot link under present label and then select open web chat in new window link at bottom.
3.Navigate to white label experience chat page and type Document-word word in type your message text box.
4.Navigate to the search results.
5.Observe that, Luminosity contrast ratio is of time text which is less than 4.5:1

### What do you expect?

The time text "6 minute ago" should have a color contrast ratio of at least 4.5:1 against its background to ensure that it is accessible to all users, including those with visual impairments.

### What actually happened?

Luminosity contrast ratio is 4.1:1 of time text which is less than 4.5:1.

### Do you have any screenshots or recordings to repro the issue?

<img width="977" height="584" alt="Image" src="https://github.com/user-attachments/assets/2992a0bc-e547-42be-a8a3-7b7edce83a46" />

### WCAG reference

https://www.w3.org/WAI/WCAG21/Understanding/contrast-minimum

### Adaptive Card JSON

```json

```
